### PR TITLE
Update DAOBot to implement QV & Weighted voting.

### PR DESCRIPTION
**Pull request by idiom-bytes**
As defined inside of [this ticket](https://github.com/oceanprotocol/oceandao/issues/16).

This PR includes all code required to configure:
`Vote Type`: single-choice, quadratic, weighted
`Ballot Type`: Batch (N Proposals -> 1 Ballot) or Granular (N Proposals -> N Ballots)

This makes it so that:
- DAOBot can be easily configured per-round.
- Round configurations are easily understood via Airtable views.

This PR include: 
- [x] Code required to update Airtable
- [x] Code required to update GSheets

I have updated the [Funding Rounds] table inside of Airtable Prod to include the configuration as found inside of my [Dev Airtable environment](https://airtable.com/shruIHermw4LiMs2x), in preparation for the merge of this PR.

### CODE REVIEW

File: src/snapshot/submit_snapshot_accepted_proposals_airtable.js [hunk 1]
```diff
@@ -71,4 +67,60 @@ const submitProposalsToSnapshot = async (roundNumber) => {
     }
 }
 
-module.exports = {submitProposalsToSnapshot};
+const submitProposalsToSnaphotBatch = async (roundNumber) => {
+    try {
```
[the remark this answers is not in the document]
Comment (idiom-bytes) [reply src/snapshot/submit_snapshot_accepted_proposals_airtable.js:71]: Thanks for "catching" that Tim! ;)

I'm not sure why I implemented it this way, I'll restructure the function to only have a single one so it doesn't introduce any weird things during error handling. Cheers!

**Comment by idiom-bytes** [comment]
So, I believe this is ready to merge. @AnaLoznianu can you please ask someone from the team to take ownership of this? I'll be reviewing progress on Mon/Tues. Cheers!

**Comment by AnaLoznianu** [comment]
Hi Tim, this PR started as a QV implementation but in the meantime, we decided that until we have time to implement Sybil protection, to adjust it for weighted voting. It adds some support for QV, but it focuses on Weighted voting that we will have for Round 11.

**Review by KatunaNorbert — APPROVED** [review]
Looks good!

**Review by claudiaHash — APPROVED** [review]
Good to go!

**Pull request merged**